packages/flutter/lib/src/material/radio_theme.dart

```diff
@@ -0,0 +1,228 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'material_state.dart';
+import 'theme.dart';
+import 'theme_data.dart';
+
+/// Defines default property values for descendant [Radio] widgets.
+///
+/// Descendant widgets obtain the current [RadioThemeData] object using
+/// `RadioTheme.of(context)`. Instances of [RadioThemeData] can be customized
+/// with [RadioThemeData.copyWith].
+///
+/// Typically a [RadioThemeData] is specified as part of the overall [Theme]
+/// with [ThemeData.radioTheme].
+///
+/// All [RadioThemeData] properties are `null` by default. When null, the
+/// [Radio] will use the values from [ThemeData] if they exist, otherwise it
+/// will provide its own defaults based on the overall [Theme]'s colorScheme.
+/// See the individual [Radio] properties for details.
+///
+/// See also:
+///
+///  * [ThemeData], which describes the overall theme information for the
+///    application.
+@immutable
+class RadioThemeData with Diagnosticable {
+  /// Creates a theme that can be used for [ThemeData.radioTheme].
+  const RadioThemeData({
+    this.fillColor,
+    this.mouseCursor,
+    this.materialTapTargetSize,
+    this.visualDensity,
+    this.focusColor,
+    this.hoverColor,
+    this.splashRadius,
+  });
+
+  /// {@macro flutter.material.radio.fillColor}
+  ///
+  /// If specified, overrides the default value of [Radio.fillColor].
+  final MaterialStateProperty<Color?>? fillColor;
+
+  /// {@macro flutter.material.radio.mouseCursor}
+  ///
+  /// If specified, overrides the default value of [Radio.mouseCursor].
+  final MouseCursor? mouseCursor;
+
+  /// {@macro flutter.material.radio.materialTapTargetSize}
+  ///
+  /// If specified, overrides the default value of
+  /// [Radio.materialTapTargetSize].
```

Comment:
nit: ensure the order of theme properties matches that of `Radio` constructor

Reply:
Done!